controls/srg_gpos/SRG-OS-000073-GPOS-00041.yml

```diff
@@ -9,8 +9,7 @@ controls:
             - set_password_hashing_algorithm_passwordauth
             - set_password_hashing_algorithm_systemauth
             - set_password_hashing_min_rounds_logindefs
-            - var_password_hashing_algorithm_pam=yescrypt
-            - set_password_hashing_yescrypt_cost_factor_logindefs
-            - var_password_pam_unix_rounds=5
-            - var_password_yescrypt_cost_factor_login_defs=5
+            - var_password_hashing_algorithm_pam=sha512
+            - set_password_hashing_min_rounds_logindefs
+            - var_password_pam_unix_rounds=100000
```

Comment:
There is `var_password_pam_unix_rounds=5` in `controls/srg_gpos/SRG-OS-000730-GPOS-00190.yml` which overrides this and causes the profile stability test to fail.